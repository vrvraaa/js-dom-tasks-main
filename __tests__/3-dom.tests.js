import '@testing-library/jest-dom';
import fs from 'fs';
import path from 'path';
import prettier from 'prettier';

const options = {
  parser: 'html',
  htmlWhitespaceSensitivity: 'ignore',
  tabWidth: 4,
};

test('scripts count', async () => {
  const initHtml = fs.readFileSync(path.join('public', '3-index.html')).toString();
  document.documentElement.innerHTML = initHtml;
  await import('../solutions/3-dom.js');
  const body = prettier.format(document.body.innerHTML, options);
  expect(body).toMatchSnapshot();
});