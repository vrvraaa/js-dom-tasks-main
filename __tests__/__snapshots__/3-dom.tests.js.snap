// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scripts count 1`] = `
"<p>I do not fear computers.</p>
<p>
    Computer Science is no more about computers than astronomy is about
    telescopes.
</p>
<p>The computer was born to solve problems that did not exist before.</p>
"
`;
